unity/Assets/Scripts/BaseFPSAgentController.cs

```diff
@@ -1390,7 +1390,6 @@ public virtual MetadataWrapper generateMetadataWrapper()
             metaMessage.lastAction = lastAction;
             metaMessage.lastActionSuccess = lastActionSuccess;
             metaMessage.errorMessage = errorMessage;
-            metaMessage.actionReturn = this.actionReturn;
```

Comment:
Should we still keep this? and set it to `actionReturns[0]` for the first agent to not break client code we have all around where we read `actionReturn` on the metadata to do certain things?

Reply:
We still pass actionreturn values as a separate form parameter and copy this in to the correct place in the metadata in server.py